src/common/njord.ts

```diff
@@ -146,27 +162,30 @@ export const transferCores = createAuthProtectedFn(
 
     const njordClient = getNjordClient();
 
-    const transferResult = await garmNjordService.execute(async () => {
-      if (!transaction.id) {
-        throw new Error('No transaction id');
-      }
+    if (!transaction.id) {
+      throw new Error('No transaction id');
+    }
 
-      if (!transaction.senderId) {
-        throw new Error('No sender id');
-      }
+    if (!transaction.senderId) {
+      throw new Error('No sender id');
+    }
 
-      const { results } = await njordClient.transfer({
+    const senderId = transaction.senderId;
+    const receiverId = transaction.receiverId;
+
+    const response = await garmNjordService.execute(async () => {
+      const response = await njordClient.transfer({
```

Comment:
Couldn't we just return this directly?

Reply:
We want to await because if you just return `execute` will not catch promise rejection.